cmake/configs/nuttx_mindpx-v2_default.cmake

```diff
@@ -94,6 +94,7 @@ set(config_module_list
 	modules/attitude_estimator_q
 	modules/position_estimator_inav
 	modules/local_position_estimator
+	modules/landing_target_estimator
```

Comment:
@bkueng Is this explicitly needed for this PR? I'm not proposing to make a separate PR just want to understand.

Reply:
Yes, there's a parameter dependency from LPE. The commit message contains the details: https://github.com/PX4/Firmware/pull/8789/commits/70db3d2c5582e302a7bd638f28d823870d65dac3